subannual/months.yaml

```diff
@@ -0,0 +1,37 @@
+# List of months
+
+January:
+   duration: 31. / 365
```

Comment:
Should make much of a difference for the actual content, but the additional '.' bothers me here

Reply:
thanks, fixed!